consent-ws/src/test/java/org/broadinstitute/consent/http/VoteDataRequestTest.java

```diff
@@ -63,7 +63,7 @@ public void setup() throws Exception {
         MongoDBRuntime runtime = MongoDBRuntime.getDefaultInstance();
 
         // Creating MongodbExecutable
-        mongodExe = runtime.prepare(new MongodConfig(Version.V2_1_2, 37017, Network.localhostIsIPv6()));
+        mongodExe = runtime.prepare(new MongodConfig(Version.V2_1_2, 37017, false, "127.0.0.1"));
```

Comment:
This test was failing in amazon because it was getting the host name from the host. In amazon, that wasn't giving us back the right value. This is more consistent with the other tests that hit mongo and don't error out in amazon.